src/components/AttachmentPicker/launchCamera/launchCamera.ios.ts

```diff
@@ -29,4 +26,5 @@ export default function launchCameraIOS(options, callback) {
                 errorCode: error.errorCode || 'others',
             });
         });
-}
+};
+export default launchCameraIOS;
```

Comment:
If a module exports something as a default, then it should be exporting something with the same name as the file. Can you please change this to export `launchCamera` instead, please?

Reply:
I suggest importing `launchCamera` with an alias name in that case. Maybe `launchCameraImagePicker`